layers/meta-resin-edison/recipes-core/images/resin-image-initramfs.bbappend

```diff
@@ -0,0 +1 @@
+IMAGE_ROOTFS_MAXSIZE = "16384"
```

Comment:
Basically what I'm interested in is to not break resinhup which needs at least free space as the biggest file available in the boot partition because it does atomic copy operations.